NuGet-Packages.md

```diff
@@ -2,25 +2,21 @@ NuGet is a standard package manager for .NET applications which is built into Vi
-
-
-
-| **NuGet Package Name**      | **Description**   |
-| :------------- | :----------: |
-|  Microsoft.Toolkit | .NET Standard NuGet package containing common code   |
-| Microsoft.Toolkit.HighPerformance   | .NET Standard and .NET Core NuGet package with performance-oriented helpers, extensions, etc |
-| Microsoft.Toolkit.MVVM | (In Preview) .NET Standard MVVM Library, [read more here](https://aka.ms/MVVMToolkit) |
-|  Microsoft.Toolkit.Parsers | 	.NET Standard NuGet package containing cross-platform parsers, such as Markdown   |
-|  Microsoft.Toolkit.Services | (Deprecating) .NET Standard NuGet package containing cross-platform services helpers, such as LinkedIn, Microsoft Graph, Twitter and more   |
-|  Microsoft.Toolkit.Uwp | Main NuGet package includes code only helpers such as Color conversions, Storage file handling, a Stream helper class, etc.   |
-|  Microsoft.Toolkit.Uwp.Notifications | 	Notifications Package - Generate tile, toast, and badge notifications for Windows 10 via code. Includes IntelliSense support to avoid having to use the XML syntax   |
-|  Microsoft.Toolkit.Uwp.UI | UI Packages - XAML converters, Visual tree extensions, State Triggers, and other extensions and helpers for your XAML UI   |
-|  Microsoft.Toolkit.Uwp.UI.Animations | 	Animations and Composition behaviors such as Blur, Fade, Rotate, etc.   |
-|  Microsoft.Toolkit.Uwp.UI.Controls | 	XAML Controls such as RadialGauge, RangeSelector, etc. |
-|  Microsoft.Toolkit.Uwp.UI.Controls.DataGrid | XAML DataGrid control   |
-|  Microsoft.Toolkit.Uwp.UI.Controls.Layout | XAML layout controls such as WrapLayout, StaggeredLayout, etc.   |
-|  Microsoft.Toolkit.Uwp.UI.Lottie | Library for rendering Adobe AfterEffects animations natively in Windows apps   |
-|  Microsoft.Toolkit.Uwp.UI.Media | Brushes, Win2D/Composition effects, and helpers to create visual effects   |
-|  Microsoft.Toolkit.Uwp.Connectivity | API helpers such as BluetoothLEHelper and Networking   |
-|  Microsoft.Toolkit.Uwp.DeveloperTools | 	XAML user controls and services to help developer building their app   |
\ No newline at end of file
+| **Old NuGet Package Name**      | **New NuGet Package Name** | **Description** |
+| :-------------: | :----------: | :----------: |
+|  Microsoft.Toolkit | | .NET Standard NuGet package containing common code   |
+| Microsoft.Toolkit.HighPerformance   | CommunityToolkit.HighPerformance | .NET Standard and .NET Core NuGet package with performance-oriented helpers, extensions, etc |
+| Microsoft.Toolkit.MVVM | CommunityToolkit.MVVM | (In Preview) .NET Standard MVVM Library, [read more here](https://aka.ms/MVVMToolkit) |
+|  Microsoft.Toolkit.Parsers | 	| .NET Standard NuGet package containing cross-platform parsers, such as Markdown   |
+|  Microsoft.Toolkit.Services | | (Deprecating) .NET Standard NuGet package containing cross-platform services helpers, such as LinkedIn, Microsoft Graph, Twitter and more   |
```

Comment:
```suggestion
```
Can probably just remove this one since it's old? The Parsers are the same, right?